elide-core/src/main/java/com/yahoo/elide/core/pagination/Pagination.java

```diff
@@ -90,21 +87,32 @@ public static Pagination parseQueryParams(final MultivaluedMap<String, String> q
             return DEFAULT_PAGINATION;
         }
 
-        boolean isPageBased = pageData.containsKey(PaginationKey.page);
-        int limit = !pageData.containsKey(PaginationKey.limit) ? DEFAULT_PAGE_SIZE : pageData.get(PaginationKey.limit);
-        if (limit < 0) {
-            throw new InvalidValueException("page[size] and page[limit] must contain positive values.");
-        }
 
-        if (isPageBased && !pageData.containsKey(PaginationKey.offset)) {
-            int page = pageData.get(PaginationKey.page);
-            pageData.put(PaginationKey.offset, (page > 0 ? page - 1 : 0) * limit);
-        }
-        if (!pageData.containsKey(PaginationKey.offset)) {
-            pageData.put(PaginationKey.offset, 0);
+        if (pageData.containsKey(PaginationKey.limit) || pageData.containsKey(PaginationKey.offset)) {
```

Comment:
Are combinations like limit and size legal?  Should we throw an error when specified?